processor/spanmetricsprocessor/config_test.go

```diff
@@ -89,9 +94,21 @@ func TestLoadConfig(t *testing.T) {
 					MetricsExporter:         tc.wantMetricsExporter,
 					LatencyHistogramBuckets: tc.wantLatencyHistogramBuckets,
 					Dimensions:              tc.wantDimensions,
+					AggregationTemporality:  tc.wantAggregationTemporality,
 				},
 				cfg.Processors[config.NewComponentID(typeStr)],
 			)
 		})
 	}
 }
+
+func TestGetAggregationTemporality(t *testing.T) {
```

Comment:
would this be redudant?

Reply:
I don't think so, what makes you say that? I'm just unit testing the function that converts the user config into a `pdata.MetricsAggregationTemporality`